Sources/HummingbirdCore/Server/HTTP/HTTP1Channel.swift

```diff
@@ -22,15 +22,48 @@ import NIOHTTPTypesHTTP1
 public struct HTTP1Channel: ServerChildChannel, HTTPChannelHandler {
     public typealias Value = NIOAsyncChannel<HTTPRequestPart, HTTPResponsePart>
 
+    /// HTTP1Channel configuration
+    public struct Configuration: Sendable {
+        /// Additional channel handlers to add to channel
```

Comment:
Can we document where these handlers are added?

Reply:
"Additional channel handlers to add to channel between HTTP part decode and HTTP request processing"?

Reply:
"Additional channel handlers to add to channel after HTTP part **decoding** and before HTTP request processing"

That sounds good

Reply:
Updated comment